Pear/Controller/MatchViewController.swift

```diff
@@ -268,6 +290,14 @@ class MatchViewController: UIViewController {
     private func setupMatchSummaries() {
         guard let pair = pair else { return }
 
+        var pronoun: String
+        if pair.pronouns == "She/Her/Hers" {
+            pronoun = "She"
+        } else if pair.pronouns == "He/Him/His" {
+            pronoun = "He"
+        } else {
+            pronoun = "They"
+        }
```

Comment:
I think this would be more appropriate as a switch statement. Also, I think you can use `let` instead of `var`.